### DISCUSSION TITLE
Use @ to mention a person

### DISCUSSION THREAD

**Discussion by Shawnxkwang** [outline/rich-markdown-editor#376] [Ideas]
It'll be very helpful if we can @ a person with this editor just like other markdown editors

**Comment by tony** [comment]
I haven't got this operating (yet) with rich-markdown-editor

There are three ProseMirror plugins I could find for this that could potentially provide a base: 

1. https://github.com/remirror/remirror/tree/master/packages/prosemirror-suggest
2. https://github.com/joelewis/prosemirror-mentions
3. https://github.com/quartzy/prosemirror-suggestions

**Reply by Shawnxkwang** [reply]
Are you able to @ a person successfully with any of these plugins? I found it's very hard to integrate prosemirror plugs to rich-markdown-editor @tony

**Reply by tony** [reply]
@Shawnxkwang We ended up picking a totally different editor (non-prosemirror based).  This is still an open issue that needs a solution

**Reply by tommoor** [reply]
If it's easier to build in I'd accept a PR for this functionality
